### DISCUSSION TITLE
Real-World Workflows for QuiVer Benchmarks

### DISCUSSION THREAD

**Discussion by mweidling** [OCR-D/quiver-benchmarks#4] [Q&A]
OCR-D QuiVer Benchmarks will be a tool for the community to monitor the quality and performance of OCR-D workflows on different kinds of data. Our goal is to help users decide which workflow(s) to choose for their own data (use case 1) and check how good their own workflows perform on our sample or their custom data (use case 2).

Currently we only provide the [workflows recommended on the OCR-D website](https://ocr-d.de/en/workflows#recommendations) for our use case 1 (in which the coordination project provides sample data and executes said workflows on each of these data sets). We are aware that this is just a starting point; The workflows given have their flaws and are unrealistic in the sense that they are not used in production as far as we are aware of.

Since we want QuiVer to be as useful as possible for all of our users we would like to ask you to share your own real-world / production workflows that you use for processing your data and tell us what kind of material you usually process and models you use. We will then integrate your workflows into QuiVer and share the results with everyone via the QuiVer web application.

Thanks a lot in advance!

**Comment by M3ssman** [comment]
OCR-D Phase III Implementationproject `ODEM` Workflow Universitäts-u. Landesbibliothek Sachsen-Anhalt
for all Monographs/Volumes at https://opendata.uni-halle.de/handle/1981185920/31824
(currently 50% ocrized)

[ulb-odem.zip](https://github.com/OCR-D/quiver-benchmarks/files/11100975/ulb-odem.zip)
